### PR TITLE
Logger hotfix

### PR DESCRIPTION
**Pull request by dsvoronin**
see commit messages

### CODE REVIEW

**Comment by sergioboi** [comment]
How did you merge it without approval?

File: subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/test/AbstractLaunchRule.kt [hunk 1]
```diff
@@ -27,13 +30,12 @@ abstract class AbstractLaunchRule : TestRule {
...
     private inner class LaunchStatement(private val base: Statement) : Statement() {
 
-        @SuppressLint("LogNotTimber")
         override fun evaluate() {
             try {
                 beforeTest()
                 base.evaluate()
             } catch (t: Throwable) {
-                Log.e(tag, "Error during test completion - $t - ${t.message}")
+                logger.critical("Error during test completion", t)
```
Comment (sergioboi) [inline subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/test/AbstractLaunchRule.kt:38]: Looks strange to catch, log and throw we simple could delete this block

**Comment by sergioboi** [comment]
It's not comfortable to read PR intention from commit messages

**Comment by eugene-krivobokov** [comment]
> How did you merge it without approval?

This is a PR into a custom branch, not develop.

**Pull request merged**